spec/cypress/integration/signup.spec.js

```diff
@@ -0,0 +1,103 @@
+import faker from 'faker'
+import { createSelector } from '../utils'
+
+const { name, internet, phone, company } = faker
+const firstName = name.firstName()
+const fullName = name.findName(firstName)
+const email = internet.email(firstName)
+// faker only allows for length and memorable options on its internet password object, so I can't guarantee there will be a number as well, so I'm hardcoding this
+const password = 'testpass1'
```

Comment:
I ran into this issue when running the Cypress tests a few times. How about using `random.alphaNumeric(15)`?